### PR TITLE
feat:  add new sphinx-theme extension with pyaedt for docs link.

### PR DESCRIPTION
**Pull request by Revathyvenugopal162**
- [x] Add the extension to  show the github source
- [x] change the occurrence of pyansys/pyaedt to ansys/pyaedt

### CODE REVIEW

**Comment by Revathyvenugopal162** [comment]
WIP

File: .github/workflows/build_documentation.yml [hunk 1]
```diff
@@ -94,9 +94,9 @@ jobs:
...
-#        with:
-#          name: Documentation
-#          path: doc/_build/html
-#          retention-days: 7
+      - name: Upload HTML documentation artifact
+        uses: actions/upload-artifact@v3
+        with:
+          name: documentation-html
+          path: doc/_build/html
+          retention-days: 7
```
Comment (Revathyvenugopal162) [inline .github/workflows/build_documentation.yml:102]:
```suggestion
#      - name: Upload Documentation
#        uses: actions/upload-artifact@v3
#        with:
#          name: Documentation
#          path: doc/_build/html
#          retention-days: 7
```

**Review by MaxJPRey — APPROVED** [review]
LGTM.

**Comment by MaxJPRey** [comment]
Thanks for replacing all the former `pyansys` reference.

**Pull request merged**